roles/cloud/files/eucalyptus-cloud-https-import

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+# Import HTTPS key/certificate from PEM files
+set -eo pipefail
+
+KEY_ALIAS="eucaconsole"
+PEM_KEY="/etc/pki/tls/private/eucaconsole.key"
+PEM_CERTS="/etc/pki/tls/certs/eucaconsole.crt"
+
+while (( "$#" )); do
+  IMPORT_ARG="$1"
+  case "${IMPORT_ARG}" in
+    --alias)
+      shift
+      KEY_ALIAS="$1"
+      ;;
+    --key)
+      shift
+      PEM_KEY="$1"
+      ;;
+    --certs)
+      shift
+      PEM_CERTS="$1"
+      ;;
+    *)
+      echo -e "Usage:\n\n\teucalyptus-cloud-https-import [--alias ALIAS] [--key FILE] [--certs FILE]\n"
```

Comment:
--key and --cert should be in [] since they are not optional.

Reply:
All parameters are optional.